sw/device/lib/crypto/drivers/hmac.c

```diff
@@ -25,7 +25,7 @@ void hmac_sha256_init(void) {
   // Digest is little-endian by default.
   reg = bitfield_bit32_write(reg, HMAC_CFG_DIGEST_SWAP_BIT, false);
   // Enable endian swap since our inputs are little-endian.
```

Comment:
This comment doesn't make sense any more, maybe:
```suggestion
  // Message is little-endian by default.
```